core/src/main/java/de/danoeh/antennapod/core/util/comparator/ChapterStartTimeComparator.java

```diff
@@ -8,13 +8,7 @@
 
 	@Override
 	public int compare(Chapter lhs, Chapter rhs) {
-		if (lhs.getStart() == rhs.getStart()) {
-			return 0;
-		} else if (lhs.getStart() < rhs.getStart()) {
-			return -1;
-		} else {
-			return 1;
-		}
+		return Long.compare(lhs.getStart(), rhs.getStart());
```

Comment:
Long.compare was added in API 19 but we support API 16+

Reply:
That's not an issue here, actually. What I think you're referring to is [this](https://developer.android.com/reference/java/lang/Long#compare(long,%20long) compare specifically, which isn't what's being used here.

https://stackoverflow.com/questions/43045966/long-compare-works-on-androidapi19-despite-docu-says-it-should-not

I'm not getting any lint warning from it.